packages/components/src/spec-distributions.json

```diff
@@ -32,89 +32,152 @@
       }
     }
   ],
-
   "scales": [
     {
       "name": "xscale",
       "type": "pow",
-      "exponent": { "signal": "xscale ? 0.1 : 1" },
+      "exponent": {
+        "signal": "xscale ? 0.1 : 1"
+      },
       "range": "width",
       "zero": false,
       "nice": false,
       "domain": {
         "fields": [
-          { "data": "con", "field": "x" },
-          { "data": "dis", "field": "x" }
+          {
+            "data": "con",
+            "field": "x"
+          },
+          {
+            "data": "dis",
+            "field": "x"
+          }
         ]
       }
     },
     {
       "name": "yscale",
       "type": "pow",
-      "exponent": { "signal": "yscale ? 0.1 : 1" },
+      "exponent": {
+        "signal": "yscale ? 0.1 : 1"
+      },
       "range": "height",
       "nice": true,
       "zero": true,
       "domain": {
         "fields": [
-          { "data": "con", "field": "y" },
-          { "data": "dis", "field": "y" }
+          {
+            "data": "con",
+            "field": "y"
+          },
+          {
+            "data": "dis",
+            "field": "y"
+          }
         ]
       }
     }
   ],
-
-  "axes": [{ "orient": "bottom", "scale": "xscale", "tickCount": 20 }],
-
+  "axes": [
+    {
+      "orient": "bottom",
+      "scale": "xscale",
+      "labelColor": "#666",
+      "tickColor": "#ddd",
+      "format": "~s",
+      "tickCount": 20
+    }
+  ],
   "marks": [
     {
       "type": "area",
-      "from": { "data": "con" },
+      "from": {
+        "data": "con"
+      },
       "encode": {
-        "enter": {
-          "tooltip": { "signal": "datum.cdf" }
-        },
         "update": {
-          "x": { "scale": "xscale", "field": "x" },
-          "y": { "scale": "yscale", "field": "y" },
-          "y2": { "scale": "yscale", "value": 0 },
+          "x": {
+            "scale": "xscale",
+            "field": "x"
+          },
+          "y": {
+            "scale": "yscale",
+            "field": "y"
+          },
+          "y2": {
+            "scale": "yscale",
+            "value": 0
+          },
           "fill": {
-            "signal": "{gradient: 'linear', x1: 1, y1: 1, x2: 0, y2: 1, stops: [ {offset: 0.0, color: '#11ac8f'}, {offset: clamp(mousex, 0, 1), color: '#11ac8f'}, {offset: clamp(mousex, 0, 1), color: '#1b6fac'}, {offset: 1.0, color: '#1b6fac'} ] }",
-            "color": "#000"
+            "signal": "{gradient: 'linear', x1: 1, y1: 1, x2: 0, y2: 1, stops: [ {offset: 0.0, color: '#4C78A8'}] }"
```

Comment:
What this did is that it allowed you to use your cursor and it would put one side in one colour and the other in a different colour. I admit that this method (making a gradient and changing the stops according to the mouse position) is a bit inelegant. Are you planning on re-introducing this functionality in a different way?

Reply:
In my testing this was fairly buggy. (Gave very wrong outputs for mixed distributions). 

I think eventually we'll want to add similar functionality, but with more feature completeness, and less buggy.

Reply:
It's buggy not because of this code, but because of #149. That being said, I would like to implement this in a different way in the future, so I'd be happy to see this go temporarily